src/models/account.model.ts

```diff
@@ -0,0 +1,38 @@
+import Setting from './setting.model';
+import {
+  Column,
+  Model,
+  Table,
+  DeletedAt,
+  CreatedAt,
+  UpdatedAt,
+  HasMany,
+  AllowNull,
+  Unique,
+} from 'sequelize-typescript';
+
+@Table
+export default class Account extends Model {
+  @Unique({
+    name: 'UniqueException',
+    msg: 'Account with this name already exists',
+  })
+  @AllowNull(false)
+  @Column
+  name: string;
+
+  @CreatedAt
+  @Column
```

Comment:
@Column decorator is not required when we add the @CreatedAt, @UpdatedAt @DeletedAt decorators

Reply:
[Done](https://github.com/ananth243/nest-starter-project/blob/732f3cf7c7bc7cf884500fc5eda0ff5aa6709c60/src/models/account.model.ts#L30)